frontend/public/components/modals/_modals.scss

```diff
@@ -60,11 +60,19 @@
   display: flex;
   flex-direction: column;
   height: 100%;
+  position: relative;
   @media(min-width: $screen-sm-min) {
     // Desktop only responsive max-height allows modal to adjust to content height and enable scroll shadows if needed.
     max-height: calc(100vh - 60px);
+    // Dropdown workaround:  use when modal content can expand (taints, tolerations)
+    &--accommodate-dropdown .modal-body-inner-shadow-covers {
+      padding-bottom: 100px;
```

Comment:
It's too bad we need to do this, but I can't think of a better fix without changing the dropdown

Reply:
The path I was going down was to add a child node within `.modal-body-inner-shadow-covers` and set a `min-height: 150px`